src/molecules/StepIndicator/index.js

```diff
@@ -13,13 +11,8 @@ function StepIndicator(props) {
     navigateToPath
   } = props;
 
-  const activeStep = steps.find(step => step.currentStepActive) || steps[0];
-
   return (
     <div className={classnames(className)}>
-      <Text tag='div' type={11} color='primary-3' font='a' spaced className={styles['active-step']}>
```

Comment:
Does the progress bar after the health navigator look OK still on iphone 5s if this is removed? That was why this was initially limited to only the active step. We may have shortened some of the breadcrumb titles though since so maybe it's not an issue anymore?

Reply:
@trevornelson good point. Since we removed the `confirmation` step, the text works fine for iphone5. maybe this might be a limitation for the StepIndicator, that on mobile it can only accept up to 4 steps? Or we could put some control flow here to render this centered text if there are more than 4 steps? I think these are questions we can punt on for now since, to me, it seems pretty low priority

Reply:
👍  as long a it looks good with the current breadcrumbs we'll be using this should be all good.